components/builder-depot-client/src/lib.rs

```diff
@@ -364,18 +360,19 @@ impl Client {
     /// * Package cannot be found
     /// * Remote Depot is not available
     /// * File cannot be created and written to
-    pub fn fetch_package<D, I, P: ?Sized>(&self,
-                                          ident: &I,
-                                          dst_path: &P,
-                                          progress: Option<D>)
-                                          -> Result<PackageArchive>
-        where P: AsRef<Path>,
+    pub fn fetch_package<D, I, P>(&self,
+                                  package: &I,
```

Comment:
I preferred the name `ident` above since it isn't a package--it's a logical representation of something to download--the ident 😄